src/touchaction.js

```diff
@@ -83,6 +83,19 @@ TouchAction.prototype = {
         var hasPanY = inStr(actions, TOUCH_ACTION_PAN_Y);
         var hasPanX = inStr(actions, TOUCH_ACTION_PAN_X);
 
+
+        if (hasNone) {
+            //do not prevent defaults if this is a tap gesture
+
+            var isTapPointer = input.pointers.length === 1;
+            var isTapMovement = input.distance === 0;
+            var isTapTouchTime = input.deltaTime < 250;
```

Comment:
Yes this has been added to keep it consistent with the author's logic for recognizing Tap gestures. Look at the TapRecognizer function for more info.